apps/src/applab/Exporter.js

```diff
@@ -571,7 +571,7 @@ export default {
       sessionId: `${getEnvironmentPrefix()}-${project.getCurrentId()}`,
       files,
       name: project.getCurrentName(),
-      sdkVersion: '32.0.0',
+      sdkVersion: '31.0.0',
```

Comment:
Bummer.  Thanks for reporting, really glad you're working directly with the Expo team on this.